content/posts/touhosu/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "touhosu"
+date: 2021-04-17T16:57:00+07:00
+summary: "A custom osu! ruleset based on Touhou Project game."
+---
+
+<!-- Wait for logo -->
+
+# status
+
+- Status : Active development
+- Last Update : 2021.502.0
+- Can import and play without error : Yes
+
+# overview
+
+- Creator : [Evast](https://github.com/EVAST9919)
+- Availability : Available on Evast's supporter on Patroen
+- Patreon : [EVAST](https://www.patreon.com/evast/posts)
+
+Touhosu is a custom osu! ruleset based on [Touhou Project game](https://en.wikipedia.org/wiki/Touhou_Project#Games). The ruleset is developed by Evast. This ruleset only available at Evast's Patreon page.
+
+# gameplay
+
+Player control a character (a character is [Reimu Hakurei](https://touhou.fandom.com/wiki/Reimu_Hakurei), main character of Touhou Project) to escape from a bullet from enemies that coming fall along with a rhythm. A character's live will gradually regen. If a character hit by a bullet, a character losts a live. Player can control a character to shoot an enemies but since creator has not add an enemies now, it's not affect. If a player want your character to move slower a player can use *Focus* to make a character moving slower.
+
+<!-- TODO: Make sure more about focus control. -->
+
+# controls
+
+Can only use keyboard to play.
+
+### default controls
+
+- Move Left - `LEFT`
+- Move Right - `RIGHT`
+- Move Up - `UP`
+- Move Down - `DOWN`
+- Shoot - `Z`
+- Focus - `SHIFT`
+
+# mods
+
+### difficulty reduction
+
+- ![Easy Icon](mod-icon/easy-mod.png) Easy
+  - Smaller bullets, more forgiving HP drain and three lives!
+  - Score Multiplier: 0.50x
+
+- ![No Fail Icon](mod-icon/no-fail-mod.png) No Fail
+  - You can't fail, no matter what.
+    - Makes it impossoble to fail.
+  - Score Multiplier: 0.50x
+
+- ![Half Time Icon](mod-icon/half-time-mod.png) Half Time
+  - Less Zoom...
+    - Decreases speed .75x.
+  - Score Multiplier: 0.30x
+  - Click the icon again for*Daycore Mod* : Whoaaaaa...
+    - Decreases speed .75x and lowers audio pitch.
+
+### difficulty increase
+
+- ![Sudden Death Icon](mod-icon/sudden-death-mod.png) Sudden Death
+  - Miss and fail.
+    - If you miss, you fail.
+
+- ![Double Time Icon](mod-icon/double-time-mod.png) Double Time
+  - Zoooooooooom...
+    - Increases speed 1.50x.
+  - Score Multiplier: 1.10x
+  - Click the icon again for `Nightcore Mod` : Uguuuuuuuu...
```

Comment:
Hmmm.. Why is this using `this style` rather than *this style*? The second was used in the Daycore explaination above.

Reply:
Oh this wrong in all rulesets file. Thank for reminding me